src/sentry/static/sentry/app/views/userFeedback/userFeedbackEmpty.tsx

```diff
@@ -108,7 +108,9 @@ class UserFeedbackEmpty extends React.Component<Props> {
                   eventName: 'User Feedback Docs Clicked',
                 })
               }
-              href="https://docs.sentry.io/enriching-error-data/user-feedback/"
+              href={`https://docs.sentry.io/platforms/${
+                this.props.projects[0].platform || 'javascript'
```

Comment:
In case there are no projects

```suggestion
                this.props.projects?.[0].platform || 'javascript'
```

Reply:
I don't think we can reach this line if `projects` is an empty array due to the check above. Am I missing something?